catalyst-gateway/bin/src/service/common/objects/voter_info.rs

```diff
@@ -9,19 +9,22 @@ use super::{delegate_public_key::DelegatePublicKey, voter_group_id::VoterGroupId
 pub(crate) struct VoterInfo {
     /// Voter's voting power.
     /// This is the true voting power, subject to minimum voting power and max cap.
-    #[oai(validator(minimum(value = "0")))]
+    // TODO - Recheck max value
+    #[oai(validator(minimum(value = "0"), maximum(value = "1000000000000000000")))]
```

Comment:
If I am not mistaken `"1000000000000000000"` stands as a `i64` maximum value right ?
Is so it makes sense to move it as a `const` value, somewhere in the `common` module

Reply:
Just a random number, but sure will do a max of `i64` apply to this. Don't think there is a max voting power right 🤔

Reply:
>Don't think there is a max voting power right 

Actually most probable that right now we already have some voting power limits, but don't remember what it is.

For now and for catalyst-gateway I think would be enough just put some reasonable maximum values as a limits for `i64` and "1000000000000000000" I think is ok, but it would be better to make as const with the appropriate name as `MAX_INTEGER` or something, to not to lose a context why we have such values.

Reply:
Somehow, variable can't be assigned to "value", so hard coded